### ISSUE TITLE
Documentation how CALL messages are handled

### ISSUE THREAD

**Issue by OrangeTux** [mobilityhouse/ocpp#55]
Currently the docs are lacking proper documentation about how routing of CALL messages work, how to use the `@on()` and `@after()` decorator should be used and how the route map is build.

**Comment by eugenio-tesio** [comment]
Do you people had an example that implement all the operations initiated by the charge point for ocpp1.6?. In chapter 4 of the Specifications it says that the operations initiated by the charger point are:

1. Authorize.
2. Boot Notification.
3. Data Transfer.
4. Diagnostics Status Notification.
5. Firmware Status Notification.
6. Heartbeat.
7. Meter Values.
8. Start Transaction.
9. Status Notification.
10. Stop Transaction.

**Comment by OrangeTux** [comment]
No, we don't have open source implementation of that in this repo.

**Comment by Bej-Technologies** [comment]
So there is no way to send a start/stop command to the charger using this repo?

**Comment by OrangeTux** [comment]
No, that's not a correct statement. It's perfectly feasible to send a RemoteStartTransaction or RemoteStopTransaction from the CSMS to a charger.  See for example [this comment.](https://github.com/mobilityhouse/ocpp/issues/86#issuecomment-620205346)

**Comment by Bej-Technologies** [comment]
Perfect! I think I just misunderstood your last comment. Now we have it up and running here, thank you for clarifying
